rpcs3/Emu/Cell/lv2/sys_game.cpp

```diff
@@ -15,11 +16,19 @@ error_code _sys_game_board_storage_read(vm::ptr<u8> buffer, u8 code)
 		return CELL_EFAULT;
 	}
 
+	u64 psid[2] = { g_cfg.sys.console_psid_high, g_cfg.sys.console_psid_low };
```

Comment:
be_t<u64> instead of u64.

Reply:
I changed it to `be_t<u64> psid[2] = { g_cfg.sys.console_psid_high, g_cfg.sys.console_psid_low };`.
But the IDE reported:
E0312 No suitable user-defined conversion from "cfg::uint<0Ui64, 18446744073709551615Ui64>" to "stx::se_t<u64, true, 8Ui64>"

Reply:
be_t<u64> psid[2] = { +g_cfg.sys.console_psid_high, +g_cfg.sys.console_psid_low };

Reply:
How about using memset()?